CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Support `CARGO_CFG_TARGET_POINTER_WIDTH`, enabling to compile rustler_sys for 32-bit systems
```

Comment:
I think this is detailing something much lower-level than we expose?

Cargo automatically exports that env var based on the `--target`. If there is no `--target` provided, then it's set to whatever the host supports.